CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ include(SetupLibsharp)
 include(SetupYamlCpp)
 
 include(SetupLIBCXXCharm)
-include(SpectreSetupPython)
+include(SetupSpectrePythonPackage)
```

Comment:
Please don't do this rename

Reply:
agreed on `SpectreSetupPythonPackage` in a call